src/main/java/io/jenkins/plugins/audit/CustomHTMLLayout.java

```diff
@@ -0,0 +1,412 @@
+        }
+
+        if (event.getContextData() != null && !event.getContextData().isEmpty()) {
+            sbuf.append("<tr><td bgcolor=\"#fbfbfb\" style=\"font-size : ").append(fontSize);
+            sbuf.append(";\" colspan=\"6\" ");
+            sbuf.append("title=\"Mapped Diagnostic Context\">");
+            sbuf.append("MDC: ").append(Transform.escapeHtmlTags(event.getContextData().toMap().toString()));
+            sbuf.append("</td></tr>").append(Strings.LINE_SEPARATOR);
+        }
+
+        final Throwable throwable = event.getThrown();
+        if (throwable != null) {
+            sbuf.append("<tr><td bgcolor=\"#993300\" style=\"color:White; font-size : ").append(fontSize);
+            sbuf.append(";\" colspan=\"6\">");
+            appendThrowableAsHtml(throwable, sbuf);
+            sbuf.append("</td></tr>").append(Strings.LINE_SEPARATOR);
+        }
+
+        return sbuf.toString();
+    }
+
+    @Override
+    /**
+     * @return The content type.
+     */
+    public String getContentType() {
+        return contentType;
+    }
+
+    private void appendThrowableAsHtml(final Throwable throwable, final StringBuilder sbuf) {
+        final StringWriter sw = new StringWriter();
+        final PrintWriter pw = new PrintWriter(sw);
+        try {
+            throwable.printStackTrace(pw);
+        } catch (final RuntimeException ex) {
+            // Ignore the exception.
+        }
+        pw.flush();
+        final LineNumberReader reader = new LineNumberReader(new StringReader(sw.toString()));
+        final ArrayList<String> lines = new ArrayList<>();
+        try {
+          String line = reader.readLine();
+          while (line != null) {
+            lines.add(line);
+            line = reader.readLine();
+          }
+        } catch (final IOException ex) {
+            if (ex instanceof InterruptedIOException) {
+                Thread.currentThread().interrupt();
+            }
+            lines.add(ex.toString());
+        }
+        boolean first = true;
+        for (final String line : lines) {
+            if (!first) {
+                sbuf.append(TRACE_PREFIX);
+            } else {
+                first = false;
+            }
+            sbuf.append(Transform.escapeHtmlTags(line));
+            sbuf.append(Strings.LINE_SEPARATOR);
+        }
+    }
+
+    private StringBuilder appendLs(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s).append(Strings.LINE_SEPARATOR);
+        return sbuilder;
+    }
+
+    private StringBuilder append(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s);
+        return sbuilder;
+    }
+
+    /**
+     * Returns appropriate HTML headers.
+     * @return The header as a byte array.
+     */
+    @Override
+    public byte[] getHeader() {
+        final StringBuilder sbuf = new StringBuilder();
+        append(sbuf, "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" ");
```

Comment:
Could you try upgrading this to just `<!DOCTYPE html>` or whatever the HTML5 preamble is?